src/constants/config.ts

```diff
@@ -5,6 +5,7 @@ export const DEFAULT_SDK_CONFIG: SdkConfig = {
   trackPages: true,
   trackReferrer: true,
   trackUTM: true,
+  trackTransactions: false,
```

Comment:
We should decide on either making the users opt-in or opt-out of features. So far it looks like the pattern is to opt-out so we should keep it this way
```suggestion
  trackTransactions: true,
```